README.md

```diff
@@ -1,24 +1,78 @@
-# README
+# Penni
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is a MVP app to keep track of your expenses, it was build with love using RoR. It includes features like authentication and integration tests for all main elements.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails
+- Tailwind CSS
+- PostgreSQL
 
-* System dependencies
+## Screenshots and demo links
 
-* Configuration
+### Splash screen
+![Spash](./srceenshots/1.png)
 
-* Database creation
+### Home screen
+![Home](./srceenshots/2.png)
 
-* Database initialization
+### Category screen
+![Home](./srceenshots/3.png)
 
-* How to run the test suite
+[Live Demo Link](https://dry-dusk-84510.herokuapp.com)
```

Comment:
- [ ] In the live demo link, after I create the first category, the app stops working. Please make the necessary changes, then update your deployed version.

Reply:
I fixed it, thanks!